src/changelog/2023/09/snapshots-devices.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Usability improvements to Device Management
+date: 2023-09-28 12:00:02.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+Four weeks ago, we began working on several [usability improvements to our Device Management](https://flowfuse.com/blog/2023/08/flowfuse-1-11-release/#usability-improvements-to-device-management-%232294). This included the ability to associate devices at the application level, allowing for easier editing of Node-RED instances on edge devices. Starting today, it will also be possible to create and manage snapshots for those standalone devices.
```

Comment:
```suggestion
Four weeks ago, we began working on several [usability improvements to our Device Management](https://flowfuse.com/blog/2023/08/flowfuse-1-11-release/#usability-improvements-to-device-management-%232294). This included the ability to associate devices at the application level, allowing for easier editing of Node-RED instances on edge devices. 

Starting today, it will also be possible to create and manage snapshots for those standalone devices.
```